language/tools/move-cli/tests/testsuite/generate_struct_layout/args.exp

```diff
@@ -0,0 +1,72 @@
+Command `sandbox publish --mode bare`:
+Command `sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M2.mv --struct C --type-args u64`:
+---
+t: U64
+b: Bool
+
+Command `sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M2.mv --struct C --type-args address`:
+---
+t: Address
+b: Bool
+
+Command `sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M2.mv --struct C --type-args vector<u8>`:
+---
+t:
+  Vector: U8
+b: Bool
+
+Command `sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M1.mv --struct B --type-args bool`:
+---
+a: Address
+c:
+  Struct:
+    t: Bool
+    b: Bool
+t: Bool
+
+Command `sandbox generate struct-layouts --module storage/0x00000000000000000000000000000001/modules/M1.mv --struct A --type-args 0x1::M1::S<u64>`:
+---
+f: U64
```

Comment:
@matbd: here are some of the examples of the YAML layouts I am currently generating. I really want to make them compatible with `serde-generate`, but these don't look the same as what I see in (e.g.) diem.yaml.

A key problem is that I can't generate YAML structs (or at least I don't know how) because serde wants the field names to be constants (whereas I'm reading them from dynamically computed layouts).